### ISSUE TITLE
What does 'aria-controls' do?

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#181]
@jobara @jhung the accessible content for the arm and leg uses the 'aria-controls' attribute.  Is this attribute necessary?  The alert elements are sort of outside the scope of the parallel DOM and in this sim the alerts are hidden most of the time.  While playing with the sim with various devices, I haven't noticed an impact from this attribute. Could it be removed?

**Comment by jobara** [comment]
@jessegreenberg, [aria-controls](https://www.w3.org/TR/wai-aria-1.1/#aria-controls) is used to create a relationship between two elements that indicates that the content of one is controlled by the other. For example, the leg and arm control the electrons. In this case we have it controlling the content of the status message.

According to "[Using the aria-controls attribute](http://tink.uk/using-the-aria-controls-attribute/)" it makes even more sense to use it when the two elements are at a greater distance apart. However, according to "[aria-controls poop](http://www.heydonworks.com/article/aria-controls-is-poop)" due to lack of support and poor implementation, it's pretty much useless at the moment.

I'd suggest that we leave it in, as support may eventually come and it is potentially useful for the AT user. However, if it is causing problems with the code, then it could be removed, but you may want to leave a comment and/or track it in a issue.

**Comment by jessegreenberg** [comment]
Thanks for clarifying @jobara, that makes sense.  We have been working on a static API that could handle aria-live called [AriaHerald](https://github.com/phetsims/balloons-and-static-electricity/blob/master/js/balloons-and-static-electricity/accessibility/AriaHerald.js), and it seems useful for that type to be able to easily set up the aria-controls relationship when the attribute is supported.

I also misunderstood how it was being used in the sim. The status element is not hidden most of the time, so if the role is supported, the element could be found.

[aria-controls poop](http://www.heydonworks.com/article/aria-controls-is-poop) gave me a chuckle. Closing.
